src/main/java/net/mcreator/ui/modgui/EnchantmentGUI.java

```diff
@@ -55,9 +56,9 @@
 
 	private final JSpinner damageModifier = new JSpinner(new SpinnerNumberModel(0, 0, 1024, 1));
 
-	private final JCheckBox isTreasureEnchantment = new JCheckBox("Check to enable");
-	private final JCheckBox isCurse = new JCheckBox("Check to enable");
-	private final JCheckBox isAllowedOnBooks = new JCheckBox("Check to enable");
+	private final JCheckBox isTreasureEnchantment = new JCheckBox(L10N.t("elementgui.enchantment.is_treasure_enchantment"));

```

Comment:
Please use "Check to enable" from the existing common translations (should be something like "elementgui.common.check_to_enable_functionality"). Same goes for the 2 entries below this one.

Reply:
it was created when I made them, but I will change them.